tst/EnergyPlus/unit/RoomAirflowNetwork.unit.cc

```diff
@@ -375,6 +379,31 @@ TEST_F(RoomAirflowNetworkTest, RAFNTest)
 
     EXPECT_NEAR(24.397538, state->dataLoopNodes->Node(2).Temp, 0.00001);
     EXPECT_NEAR(0.0024802305, state->dataLoopNodes->Node(2).HumRat, 0.000001);
+
+    // #8419
+    state->dataZoneEquip->ZoneEquipList(ZoneNum).EquipTypeEnum(1) = DataZoneEquipment::ZoneEquip::AirDistUnit;
+    state->dataRoomAirflowNetModel->InitRoomAirModelAirflowNetworkOneTimeFlagConf = true;
+    state->dataZoneAirLoopEquipmentManager->GetAirDistUnitsFlag = false;
+    state->dataDefineEquipment->AirDistUnit.allocate(1);
+    thisRAFN.numAirDistUnits = 1;
+    state->dataZoneEquip->ZoneEquipList(ZoneNum).EquipName(1) = "ADU";
+    state->dataDefineEquipment->AirDistUnit(1).Name = "ADU";
+    state->dataDefineEquipment->AirDistUnit(1).EquipName.allocate(1);
+    state->dataDefineEquipment->AirDistUnit(1).EquipName(1) = "AirTerminal";
+    state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(1).HVAC(1).Name = "AirTerminal";
+    state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(1).HVAC(1).SupplyFraction = 0.4;
+    state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(1).HVAC(1).ReturnFraction = 0.4;
+    state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(2).HVAC(1).Name = "AirTerminal";
+    state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(2).HVAC(1).SupplyFraction = 0.6;
+    state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(2).HVAC(1).ReturnFraction = 0.6;
+
+    thisRAFN.InitRoomAirModelAirflowNetwork(*state, RoomAirNode);
+    // No errorfound
+    EXPECT_NEAR(1.1824296, state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(RoomAirNode).RhoAir, 0.00001);
+    EXPECT_NEAR(1010.1746, state->dataRoomAirMod->RoomAirflowNetworkZoneInfo(ZoneNum).Node(RoomAirNode).CpAir, 0.001);
+
+    thisRAFN.numAirDistUnits = 0;
+    state->dataRoomAirflowNetModel->InitRoomAirModelAirflowNetworkOneTimeFlagConf = false;
```

Comment:
This is perfectly fine.  And no big deal, but there's no need to 'cleanup' state at the end of the unit test, it will be wiped entirely for every new unit test being run.